graylog2-server/src/main/java/org/graylog2/inputs/codecs/JsonPathCodec.java

```diff
@@ -105,6 +134,50 @@ protected String buildShortMessage(Map<String, Object> fields) {
         return shortMessage.toString();
     }
 
+    public Map<String, Object> useFlattener(String json) throws JsonTypeException {
```

Comment:
We can just name this method `flatten`. Due to overloading, the correct one will be picked based upon parameter arity and types.